worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgePlatform.java

```diff
@@ -212,17 +215,23 @@ public String getPlatformVersion() {
         return capabilities;
     }
 
-    private static final Set<SideEffect> SUPPORTED_SIDE_EFFECTS = Sets.immutableEnumSet(
+    private static final Set<SideEffect> SUPPORTED_SIDE_EFFECTS_NO_MIXIN = Sets.immutableEnumSet(
         SideEffect.VALIDATION,
         SideEffect.ENTITY_AI,
         SideEffect.LIGHTING,
         SideEffect.NEIGHBORS,
         SideEffect.EVENTS
     );
 
+    private static final Set<SideEffect> SUPPORTED_SIDE_EFFECTS = Sets.immutableEnumSet(
+        Iterables.concat(SUPPORTED_SIDE_EFFECTS_NO_MIXIN, Collections.singleton(SideEffect.UPDATE))
+    );
+
     @Override
     public Set<SideEffect> getSupportedSideEffects() {
-        return SUPPORTED_SIDE_EFFECTS;
+        return ExtendedChunk.class.isAssignableFrom(LevelChunk.class)
```

Comment:
Does this even do anything? Can mixins fail to load randomly?

Reply:
Sometimes forge will change their transformer system during the update process without giving Mixin notice/time to update for their changes, and will temporarily remove the mixin library from their runtime until it gets updated.

Reply:
It's also possible for another mod to explicitly disable our mixin - I feel a fallback is safest